Task_DEV-7/DateAndTimeChecker.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System;
+
+namespace task_DEV_7
+{
+    /// <summary>
+    /// Check date and time for valid values
+    /// </summary>
+    class DateAndTimeChecker
+    {
+        private DateAndTime dateAndTime;
+        private int[] dayInMonth = { 31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31 };
+        private const int MaxValueOfYear = 9999;
+        private const int MaxValueOfMonth = 12;
+        private const int MaxValueOfHour = 24;
+        private const int MaxValueOfMinutes = 60;
+
+        public DateAndTimeChecker(DateAndTime dateAndTime)
+        {
+            this.dateAndTime = dateAndTime;
+        }
+
+        /// <summary>
+        /// Check all elements of date and time
+        /// </summary>
```

Comment:
Method check has to return bool, not void. Check means result not only message in console